README.md

```diff
@@ -45,7 +45,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.0-SNAPSHOT'
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.0'
```

Comment:
This example is specifically showing how to use a snapshot and should not be changed.

Reply:
Maybe you can add a new example of configuring the release version of the plugin, separate from showing how to use a snapshot?

Reply:
You can also remove the snapshot repo from that config.